style.css

```diff
@@ -1,65 +1,54 @@
 body {
-    padding: 0;
-    margin: 0;
-    font-family: Arial, Helvetica, sans-serif;
-    background-color: rgb(226, 226, 226);
-}
-
-header {
-    text-align: center;
-}
-
-div.todo {
-    padding: 1em;
-}
+  padding: 0;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #e2e2e2; }
 
 .container {
-    max-width: 25em;
-    margin: 0 auto;
-    background-color: white;
-    box-shadow: 0 0 10px rgb(172, 172, 172);
-}
-
-/* New Todo Field */
-#new-todo {
+  max-width: 25em;
+  margin: 0 auto;
+  background-color: white;
+  box-shadow: 0 0 10px #acacac; }
+
+.todo {
+  padding: 1em;
+  /* New Todo Entry Field */ }
+  .todo .header {
```

Comment:
So yea don't nest, and avoid these type things too. This is probably a `.todo__header`

Same idea goes for the selections below